src/complex/Filter/Output.hpp

```diff
@@ -139,6 +156,11 @@ struct COMPLEX_EXPORT OutputActions
     deferredActions.emplace_back(std::move(action));
   }
 
+  void appendModifiedAction(const DataPath& dataPath, DataModifiedAction::ModifiedType modifiedType)
+  {
+    modifiedActions.emplace_back(DataModifiedAction{dataPath, modifiedType});
+  }
```

Comment:
```suggestion
  void appendModifiedAction(DataPath dataPath, DataModifiedAction::ModifiedType modifiedType)
  {
    modifiedActions.emplace_back(std::move(dataPath), modifiedType);
  }
```
`emplace_back` can take the arguments of the object it's constructing allowing to create the object in place in the vector.

Reply:
using push_back() instead. Clang does not allow explicit something or another.